services/web-app/components/back-to-top/back-to-top.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { UpToTop } from '@carbon/icons-react'
+import clsx from 'clsx'
+import { useCallback, useEffect, useState } from 'react'
+
+import useEventListener from '@/utils/use-event-listener'
+
+import styles from './back-to-top.module.scss'
+
+/**
+ * The `<BackToTop />` component displays in the lower right corner
+ * when the user scrolls past 300px
+ */
+const BackToTop = () => {
+  const [hidden, setHidden] = useState(true)
+
+  const scrollHandler = useCallback(() => {
+    if (window.scrollY >= 300) {
+      setHidden(false)
+    }
+    if (window.scrollY < 300) {
+      setHidden(true)
+    }
```

Comment:
```suggestion
    setHidden(window.scrollY < 300)
```
this will just set it to false if it doesn't meet the condition